Cleaned up players turn. Added comments

catcom9 opened the pull request.

catcom9 committed:
Cleaned up players turn. Added comments

ernestoperzz commented:
LET ME FINISH THE PROJECT

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: catcom9 ***@***.***>
Sent: Saturday, March 2, 2024 12:13:50 PM
To: ernestoperzz/TicTacToe ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [ernestoperzz/TicTacToe] Cleaned up players turn. Added comments (PR #6)


Merged #6<https://github.com/ernestoperzz/TicTacToe/pull/6> into main.

—
Reply to this email directly, view it on GitHub<https://github.com/ernestoperzz/TicTacToe/pull/6#event-11987278240>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BD2PLFUVJ2BJGOD4AKYSYT3YWG675AVCNFSM6AAAAABEDBSDKSVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJRHE4DOMRXHAZDIMA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Tá an ríomhphost seo faoi réir an tséanta seo leanas atá le fáil ag Séanadh Ríomhphost Ollscoil Teicneolaíochta an Atlantaigh<https://www.atu.ie/email-disclaimer>
This e-mail is subject to the following disclaimer available at Atlantic TU Email Disclaimer<https://www.atu.ie/email-disclaimer>

catcom9 commented:
I was only cleaning up my part. Dont worry
________________________________
From: ernestoperzz ***@***.***>
Sent: Saturday, March 2, 2024 12:16 PM
To: ernestoperzz/TicTacToe ***@***.***>
Cc: CATHAL COMMINS - STUDENT ***@***.***>; State change ***@***.***>
Subject: Re: [ernestoperzz/TicTacToe] Cleaned up players turn. Added comments (PR #6)


LET ME FINISH THE PROJECT

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: catcom9 ***@***.***>
Sent: Saturday, March 2, 2024 12:13:50 PM
To: ernestoperzz/TicTacToe ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [ernestoperzz/TicTacToe] Cleaned up players turn. Added comments (PR #6)


Merged #6<https://github.com/ernestoperzz/TicTacToe/pull/6> into main.

—
Reply to this email directly, view it on GitHub<https://github.com/ernestoperzz/TicTacToe/pull/6#event-11987278240>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BD2PLFUVJ2BJGOD4AKYSYT3YWG675AVCNFSM6AAAAABEDBSDKSVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJRHE4DOMRXHAZDIMA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Tá an ríomhphost seo faoi réir an tséanta seo leanas atá le fáil ag Séanadh Ríomhphost Ollscoil Teicneolaíochta an Atlantaigh<https://www.atu.ie/email-disclaimer>
This e-mail is subject to the following disclaimer available at Atlantic TU Email Disclaimer<https://www.atu.ie/email-disclaimer>

—
Reply to this email directly, view it on GitHub<https://github.com/ernestoperzz/TicTacToe/pull/6#issuecomment-1974781584>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BDSKEFVYNPE55VVDZOQBA5DYWG7JDAVCNFSM6AAAAABEDBSDKSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSNZUG44DCNJYGQ>.
You are receiving this because you modified the open/close state.Message ID: ***@***.***>
